app/src/main/java/com/yterletskyi/happyfriend/features/contacts/ui/ContactsFragment.kt

```diff
@@ -51,6 +51,22 @@ class ContactsFragment : BaseBindingFragment<FragmentContactsBinding>(
             }
         }
 
+        with(binding.etSearch) {
+            doAfterTextChanged {
+                if (it?.isEmpty() == false) {
+                    binding.btnClear.visibility = View.VISIBLE
+                } else {
+                    binding.btnClear.visibility = View.GONE
+                }
```

Comment:
oh I forgot to mention it earlier. Can we use `binding.btnClear.isVisible = it?.isEmpty() == false` ?